swagger.go

```diff
@@ -194,7 +197,7 @@ const swagger_index_templ = `<!-- HTML for static distribution bundle build -->
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Swagger UI</title>
+  <title>{{.Title}}</title>
```

Comment:
For backward compatibility please put the title in the default config.

Reply:
supported